terraform/modules/self-service/modules/cognito/cognito.tf

```diff
@@ -104,24 +104,24 @@ resource "aws_cognito_user_pool" "user_pool" {
     invite_message_template {
       email_subject = "You have been invited to collaborate on the GOV.UK Verify Manage certificates service"
       email_message = <<-EOT
-        Dear {username}
-
-        You have been invited to collaborate on the GOV.UK Verify Manage certificates service.
-
-        Sign in at ${var.domain} using the following temporary password:
-
-        {####}
-
-        You will be asked to create a new password and set up multi-factor authentication using your preferred authentication app.
-
-        Please sign in within 24 hours, otherwise the temporary password will expire.
-
-        If you miss this deadline, contact your admin to ask for another temporary password.
-
-        Thanks
-
-        The GOV.UK Verify team
-        https://www.verify.service.gov.uk/
+        Dear {username}<br>
+        <br>
+        You have been invited to collaborate on the GOV.UK Verify Manage certificates service.<br>
+        <br>
+        Sign in at ${var.domain} using the following temporary password:<br>
+        <br>
+        <strong>{####}</strong><br>
+        <br>
+        You will be asked to create a new password and set up multi-factor authentication using your preferred authentication app.<br>
+        <br>
+        <span style="#d4351c">Please sign in within 24 hours, otherwise the temporary password will expire.</span><br>
```

Comment:
That doesn't look like CSS to me. Is it supposed to be?

Reply:
No... Its straight HTML as its sent from AWS.  So there are limits to what to include in the e-mail template in terraform.

Reply:
But there is a mistake there ;-)

Reply:
I can clarify what I meant: the contents of a `style` attribute must be CSS, and this isn't.